pkg/plugin/init.go

```diff
@@ -0,0 +1,26 @@
+package plugin
+
+import (
+	"encoding/json"
+	"github.com/webhookx-io/webhookx/db/entities"
+	"github.com/webhookx-io/webhookx/pkg/plugin/types"
+	"github.com/webhookx-io/webhookx/pkg/plugin/webhookx_signature"
+)
+
+func New(name string) types.Plugin {
+	switch name {
+	case "webhookx-signature":
+		return webhookx_signature.New()
+	}
+	return nil
+}
+
+func ExecutePlugin(plugin *entities.Plugin, req *types.Request, ctx *types.Context) error {
+	instance := New(plugin.Name)
+	err := json.Unmarshal(plugin.Config, instance.Config())
+	if err != nil {
+		return err
+	}
+	instance.Execute(req, ctx)
```

Comment:
`instance` will be nil when `func New(name string) types.Plugin` didn't find a plugin implement. 
If instance is nil, `ExecutePlugin` should return nil directly.